bedrock/teach/templates/teach/smarton/tracking.html

```diff
@@ -0,0 +1,376 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "teach/smarton/base.html" %}
+
+{% block page_title %}{{ _('Get smart on tracking') }}{% endblock %}
+{% block page_desc %}{{ _('Get smart on tracking') }}{% endblock %}
+{% block body_id %}smarton-tracking{% endblock %}
+{% block body_class %}smarton theme-tracking state-default{% endblock %}
+
+{% block smarton_content %}
+<article class="main">
+
+  {{ smarton_navbar(_('Tracking'), _('Get smart on tracking'), 'https://mzl.la/TODO') }}
+
+  <section id="ask" class="section section-ask">
+    <div class="content">
+
+      <h2 class="section-title">{{ _('Ask yourself') }}</h2>
+
+      <ol class="big-list">
+        <li>
+          <p>{{ _('Do you feel in control of your personal information online?') }}</p>
+          {% include 'teach/smarton/includes/list-number-1.svg' %}
+        </li>
+        <li>
+          <p>{{ _('Do you know what a cookie is (no, not the delicious kind)?') }}</p>
+          {% include 'teach/smarton/includes/list-number-2.svg' %}
+        </li>
+        <li>
+          <p>{{ _('Have you ever wondered why some content, like ads, seems to follow you around?') }}</p>
+          {% include 'teach/smarton/includes/list-number-3.svg' %}
+        </li>
+      </ol>
+
+    </div>
+  </section>
+
+  <section id="know" class="section section-know">
+    <div class="content">
+
+      <h2 class="section-title">{{ _('What is tracking?') }}</h2>
+
+      <p class="section-intro">
+        {{ _('Every time we go online, we leave behind traces of our activity.') }}
+        {{ _('They’re often called our “digital footprints” and for good reason, because tracking is sort of like re-tracing our steps.') }}
```

Comment:
Please group the first 2 sentences into one string
```
{{ _('Every time we go online, we leave behind traces of our activity. They’re often called our “digital footprints” and for good reason, because tracking is sort of like re-tracing our steps.') }}
```